dev/reference/tools/data-migration/features/overview.md

```diff
@@ -341,6 +341,10 @@ filters:
 
 ## Column mapping
 
+> **注意：**
+>
+> 我们强烈不推荐使用 Column mapping 功能，其它解决方案请参考 [自增主键冲突处理](/dev/reference/tools/data-migration/usage-scenarios/best-practice-dm-shard.md#自增主键冲突处理)。
```

Comment:
```我们强烈不推荐使用 Column mapping 功能，其它解决方案请参考 ```
-> 
```由于 Column mapping 的使用限制较多，我们不推荐优先使用 Column mapping 功能，我们优先推荐的方案请参考```

Reply:
done in https://github.com/pingcap/docs-cn/pull/1847/commits/b4b6f130d30c824286404fa31b493c176e7a6e12.